x/smart-account/keeper/keeper.go

```diff
@@ -156,6 +156,7 @@ func (k Keeper) GetInitializedAuthenticatorForAccount(
 const FirstAuthenticatorId = 1
 
 // GetNextAuthenticatorId returns the next authenticator id
+// nit: getter that mutates - should we change name to InitializeOrGetNextAuthenticatorId?
 func (k Keeper) GetNextAuthenticatorId(ctx sdk.Context) uint64 {
```

Comment:
Note: Please feel free to commit if disagreement. If PR is approved, I will change the name pre-merge
```suggestion
// nit: getter that mutates - should we change name to InitializeOrGetNextAuthenticatorId?
func (k Keeper) GetNextAuthenticatorId(ctx sdk.Context) uint64 {
```

Reply:
This I completely agree with and I think we should change it